src/Resources/config/provider.xml

```diff
@@ -19,7 +19,7 @@
             <argument type="string">%sonata.media.thumbnail.format.default%</argument>
         </service>
         <service id="sonata.media.thumbnail.liip_imagine" class="%sonata.media.thumbnail.liip_imagine%">
-            <argument type="service" id="router"/>
+            <argument type="service" id="liip_imagine.cache.manager"/>
```

Comment:
this is loaded always? because someone could not have this service defined because it is not using liip imagine